Makefile

```diff
@@ -4,10 +4,25 @@ VERSION_PATCH = 0
 VERSION = $(VERSION_MAJOR).$(VERSION_MINOR).$(VERSION_PATCH)
 GIT_SHORT_HASH = $(shell git rev-parse --short HEAD)
 
+TOOL_NAME = xcdiff
+PREFIX = /usr/local
+INSTALL_PATH = $(PREFIX)/bin/$(TOOL_NAME)
+BUILD_PATH = .build/release/$(TOOL_NAME)
+LIB_INSTALL_PATH = $(PREFIX)/lib/xcdiff
+
+SWIFT_LIB_FILES = .build/release/AEXML.swiftdoc .build/release/AEXML.swiftmodule .build/release/Basic.swiftdoc .build/release/Basic.swiftmodule .build/release/PathKit.swiftdoc .build/release/PathKit.swiftmodule .build/release/SPMLibc.swiftdoc .build/release/SPMLibc.swiftmodule .build/release/SPMUtility.swiftdoc .build/release/SPMUtility.swiftmodule .build/release/XCDiff.swiftdoc .build/release/XCDiff.swiftmodule .build/release/XCDiffCommand.swiftdoc .build/release/XCDiffCommand.swiftmodule .build/release/XCDiffCore.swiftdoc .build/release/XCDiffCore.swiftmodule .build/release/XcodeProj.swiftdoc .build/release/XcodeProj.swiftmodule
```

Comment:
I believe copying the binary to `/us/local/bin` should suffice to get `xcdiff` running, the libraries are only needed to build it.

Reply:
You are right I removed the copy for the `SWIFT_LIB_FILES ` since it's not needed. 🙌